pkgs/development/python-modules/craft-application/default.nix

```diff
@@ -103,6 +103,10 @@ buildPythonPackage rec {
       "test_to_yaml_file"
       # Tests expecting pytest-time
       "test_monitor_builds_success"
+      # Temporary fix until new release to support Python 3.13
```

Comment:
Shouldn't we mark it broken on 3.13 instead?

Reply:
We *can* do, but from what I can tell, the library functions just fine, it's just the actual test code that's broken. WDYT?

Reply:
It that case disabling the test seems fine.